python/build-wheel-macos.sh

```diff
@@ -67,7 +67,7 @@ for ((i=0; i<${#PY_VERSIONS[@]}; ++i)); do
     $PIP_CMD install --upgrade setuptools
     # Install setuptools_scm because otherwise when building the wheel for
     # Python 3.6, we see an error.
-    $PIP_CMD install -q setuptools_scm==2.1.0
+    $PIP_CMD install -q setuptools_scm==3.1.0
```

Comment:
What was the issue?

Reply:
On macos python 2.7 it didn't support SETUPTOOLS_SCM_PRETEND_VERSION